docs/section19/19-4.md

```diff
@@ -0,0 +1,90 @@
+                - Check if the hourly values are equal between the design_cooling_multiplier_sch and the get_most_used_weekday_hourly_schedule : `if most_used_weekday_hourly_schedule[y] != x: int_lgt_pass_cooling = false`  
+                - Add 1 to y: `y = y +1`  
+
+
+        Conduct checks for the miscellaneous objects:  
+        - Get list of misc equipment objects: `misc_obj_list = list(space.miscellaneous_equipment)`      
+        - Reset misc_pass_cooling boolean variable: `misc_pass_cooling = true`   
+        - For each misc equipment object: `for misc in misc_obj_list:`  
+            - Get the multiplier schedule: `multiplier_sch = get_component_by_id(B_RMI,misc.multiplier_schedule)`  
+            - Get the design_cooling_multiplier_schedule: `design_cooling_multiplier_sch = multiplier_sch.cooling_design_day_sequence`  
+            - Get the get_most_used_weekday_hourly_schedule (this will be a list with 24 values for each hour of a day): `most_used_weekday_hourly_schedule = get_most_used_weekday_hourly_schedule(B_RMI, ASHRAE229,multiplier_sch)`  
+            - Reset y = 0: `y = 0`
+            - Check if each value in the design_cooling_multiplier_sch matches the corresponding hourly value in the get_most_used_weekday_hourly_schedule: `for x in list(design_cooling_multiplier_sch):`
+                - Check if the hourly values are equal between the design_cooling_multiplier_sch and the get_most_used_weekday_hourly_schedule : `if most_used_weekday_hourly_schedule[y] != x: misc_pass_cooling = false`  
+                - Add 1 to y: `y = y +1`  
+
+        - **Rule Assertion:** 
+        - Case 1: If all schedules pass per above then pass: `if all(inf_pass_cooling,occ_pass_cooling,int_lgt_pass_cooling,misc_pass_cooling) == true: outcome = "PASS"`  
+        - Case 2: Else, any of the schedules fail: `Else: outcome = "FAIL" and raise_message "<insert space.id> does not appear to have been modeled following the rule that for baseline cooling sizing runs in residential dwelling units, the infiltration, occupants, lighting, gas and electricity design day cooling schedules shalled be modeled using an equipment hourly schedule that is the same as the most used hourly weekday schedule from the annual simulation." `  
+
+
+
+
+**Notes/Questions:**  
+1. 90.1 definition of dwelling unit " a single unit providing complete independent living facilities for one or more persons, including permanent provisions for living, sleeping, eating, cooking, and sanitation." Not sure any other space types other than dwelling unit meet this definition. 
+2. Thought about including an applicability check at the start to see if there are any dwelling units in the building but it would involve looping through each space which is the same thing that the existing app check is doing so I left it out.
+3. Jason plans to add .cooling_design_day_sequence to the schema.  
```

Comment:
At what level would this happen? Perhpas we should instead add 48 values to each schedule object for heating desing day and cooling desing day.

Reply:
Discussed and resolved on call. Jason to implement original approach.